packages/firefox-deb/firefox-deb.pacscript

```diff
@@ -1,9 +1,10 @@
 name="firefox-deb"
 gives="firefox"
-version="105.0"
+version="107.0.1"
 replace="${gives}"
-url="http://launchpadlibrarian.net/624606759/firefox_${version}+build2-0ubuntu0.20.04.1_amd64.deb"
+#url="http://launchpadlibrarian.net/624606759/firefox_${version}+build2-0ubuntu0.20.04.1_amd64.deb"
```

Comment:
What's this comment for?

Reply:
In future if launchpadlibrarian is needed, the number, id number is necessary